reports-backend/src/main/java/org/jboss/da/reports/backend/impl/VersionFinderImpl.java

```diff
@@ -61,18 +64,30 @@ public String getBestMatchVersionFor(GAV gav, List<String> availableVersions) {
         return findBiggestMatchingVersion(gav, availableVersions);
     }
 
+    private List<String> getBuiltVerionsFor0(List<String> allVersions) {
```

Comment:
The 0 is intentional. An extracted method, which contains main part of the logic